Library/Homebrew/cask/lib/hbc/audit.rb

```diff
@@ -246,13 +246,25 @@ def check_latest_with_appcast
       add_warning "Casks with an appcast should not use version :latest"
     end
 
-    def check_github_releases_appcast
+    def check_hosting_with_appcast
       return if cask.appcast
+      check_github_releases_appcast
+      check_sourceforge_appcast
+    end
+
+    def check_github_releases_appcast
       return unless cask.url.to_s =~ %r{github.com/([^/]+)/([^/]+)/releases/download/(\S+)}
 
       add_warning "Cask uses GitHub releases, please add an appcast. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md"
     end
 
+    def check_sourceforge_appcast
+      return if cask.version.latest?
+      return unless cask.url.to_s =~ %r{sourceforge.net/(\S+)}
+
+      add_warning "Cask hosted on SourceForge, please add an appcast. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md"
```

Comment:
Minor language tweak: `Download is hosted on SourceForge` to avoid confusion.

(Adding the `is` would be just for consistency with the GitHub warning, which is also a complete sentence.)